python/runtime/cudaq/algorithms/py_resource_count.cpp

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2025 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#include "runtime/cudaq/platform/py_alt_launch_kernel.h"
+#include "utils/LinkedLibraryHolder.h"
+#include "utils/OpaqueArguments.h"
+#include "mlir/Bindings/Python/PybindAdaptors.h"
+#include "mlir/CAPI/IR.h"
+#include "mlir/Dialect/Func/IR/FuncOps.h"
+
+#include <fmt/core.h>
+#include <pybind11/functional.h>
+#include <pybind11/stl.h>
+
+namespace py = pybind11;
+
+namespace cudaq {
+void pyAltLaunchKernel(const std::string &, MlirModule, OpaqueArguments &,
```

Comment:
I think we have that defined in "runtime/cudaq/platform/py_alt_launch_kernel.h" already

Reply:
Yes, that looks like an unnecessary forward reference.